packages/docs/pages/apps/frappe/connection.md

```diff
@@ -0,0 +1,11 @@
+# Frappe / ERPNext
+
+:::info
+This page explains the steps you need to follow to set up the Frappe / ERPNext
+connection in Automatisch. If any of the steps are outdated, please let us know!
+:::
+
+1. Setup a new `OAuth Client` on your Frappe / ERPNext instance by following [this guide](https://docs.frappe.io/framework/user/en/guides/integration/how_to_set_up_oauth#add-a-client-app). Copy **OAuth Redirect URL** from Automatisch to **Redirect URIs** and **Default Redirect URI** fields in Frappe's OAuth Client document.
+1. Set your Site URL in the Automatisch connection dialog.
```

Comment:
Update the list position numbering.

Reply:
Sure. Actually this is flexible in terms of reordering, so used this, but good idea to stick to convention.